### PR TITLE
Add more tests for block search

### PR DESCRIPTION
**Pull request by PierreRambaud**
This change is [Reviewable](https://reviewable.io/reviews/prestashop/ps_facetedsearch/71)

### CODE REVIEW

**Comment by misiuplus** [comment]
Hello, can I use it on manufactur (brand) page?

**Comment by PierreRambaud** [comment]
@misiuplus Hi, what do you mean? :thinking:

File: tests/php/FacetedSearch/Filters/BlockTest.php [hunk 1]
```diff
@@ -396,34 +319,75 @@ public function testGetFiltersBlockWithoutWeight()
...
+        $this->mockTranslator([
             ['Availability', [], 'Modules.Facetedsearch.Shop', 'Quantity'],
             ['Not available', [], 'Modules.Facetedsearch.Shop', 'Not available'],
             ['In stock', [], 'Modules.Facetedsearch.Shop', 'In stock'],
-        ];
+        ]);
+        $this->mockLayeredCategory([['type' => 'quantity', 'filter_show_limit' => false, 'filter_type' => 1]]);
+
+        $adapterInitialMock = $this->getMockBuilder(MySQL::class)
+                            ->setMethods(['count', 'valueCOunt'])
```
Comment (matks) [inline tests/php/FacetedSearch/Filters/BlockTest.php]: `valueCOunt` => `valueCount` ?

Comment (PierreRambaud) [reply tests/php/FacetedSearch/Filters/BlockTest.php]: Thanks, nice catch!

**Comment by misiuplus** [comment]
> 
> 
> @misiuplus Hi, what do you mean? 🤔

Hello, I would like use faceted search in brand page. Is it possible?

**Pull request merged**